.github/workflows/knative-style.yaml

```diff
@@ -106,7 +106,7 @@ jobs:
         if: steps.golangci_configuration.outputs.files_exists == 'true'
         uses: golangci/golangci-lint-action@v2
         with:
-          version: v1.42
+          version: v1.43
```

Comment:
This change is taken from #99

Reply:
I hope that won't break update actions :see_no_evil:

Reply:
I think it won’t.